react-native/ReactNativeFlipperExample/ios/ReactNativeFlipperExample/AppDelegate.m

```diff
@@ -12,35 +12,17 @@
 #import <React/RCTRootView.h>
 
 #ifdef FB_SONARKIT_ENABLED
-#import <FlipperKit/FlipperClient.h>
-#import <FlipperKitLayoutPlugin/FlipperKitLayoutPlugin.h>
-#import <FlipperKitNetworkPlugin/FlipperKitNetworkPlugin.h>
-#import <FlipperKitReactPlugin/FlipperKitReactPlugin.h>
-#import <FlipperKitUserDefaultsPlugin/FKUserDefaultsPlugin.h>
-#import <SKIOSNetworkPlugin/SKIOSNetworkAdapter.h>
-
-static void InitializeFlipper(UIApplication* application) {
```

Comment:
Why this is removed ? I will prefer a c function over a class initialization and a method call.

Reply:
I'm not familar with Obj-C ecosystem so I'm prepared to change this if needed, this is just a initial implementation of this change I'm proposing.

But you can keep the current static InitializeFlipper inside your `AppDelegate.m` as a custom Flipper initialization

Reply:
So can you revert this change ?